deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/resource/EnvDeploys.java

```diff
@@ -227,6 +227,7 @@ public Response create(
                     buildBean.getArtifact_url()));
         }
         // if the stage is not allowed (allow_private_build)
+        // TODO: remove this code once is_sox db column fully populated (before merging this)
```

Comment:
Should have followup jira for tracking this?

Reply:
Good idea, done.